util/syn_yosys.sh

```diff
@@ -10,59 +10,139 @@
 # The following tools are required:
 #  - sv2v: SystemVerilog-to-Verilog converter from github.com/zachjs/sv2v
 #  - yosys: synthesis tool from github.com/YosysHQ/yosys
+#  - Cadence Conformal
 #
 # Usage:
-#   syn_yosys.sh 2>&1 | tee syn.std
+#   ./syn_yosys.sh |& tee syn.std
 
 #-------------------------------------------------------------------------
 # use fusesoc to generate files and file list
 #-------------------------------------------------------------------------
-\rm -Rf build
-fusesoc --cores-root .. run --target=sim --setup --build formal > /dev/null 2>&1
+rm -Rf build syn_out
+
+fusesoc --cores-root .. run --tool=icarus --target=syn \
+  --setup "lowrisc:systems:top_earlgrey" > /dev/null 2>&1
 
 # copy all files into directory "syn_out"
-\rm -Rf syn_out
 mkdir syn_out
-cp build/formal_0/src/*/*/*.sv build/formal_0/src/*/*/*/*.sv syn_out
+cp build/*/src/*/*.sv* build/*/src/*/*/*.sv* build/*/src/*/*/*/*.sv* syn_out
+cd syn_out
```

Comment:
This is a matter of preference, I guess, but I'd consider not changing directory. I don't think that any of the following commands actually need it. The advantage of not changing directory is just that shell scripts are easier to read (because a path means the same thing on every line).

I realise that the existing script changed directory but, since you're rewriting that bit of it, maybe this is an opportunity to improve things.

Reply:
Good point, will look into this when I create the new PR.